docs/shadow_config_spec.md

```diff
@@ -711,4 +710,14 @@ The simulated time at which to execute the process.
 Default: null  
 Type: String OR Integer OR null
 
-The simulated time at which to send a `SIGTERM` signal to the process.
+The simulated time at which to send 
+[`hosts.<hostname>.processes[*].shutdown_signal`](#hostshostnameprocessesshutdown_time)
+to the process.
+
+#### `hosts.<hostname>.processes[*].shutdown_signal`
+
+Default: "SIGTERM"
+Type: String OR Integer OR null
```

Comment:
What does a "null" signal mean?

Reply:
Hmm, yeah I think was confused here thinking that "null" meant that it didn't need to be specified by the user, but I think that's already covered by there being a `Default`.